tapToChangeMat.js

```diff
@@ -0,0 +1,44 @@
+// Script to 'tap to change' an object's material
+// Rewritten for Spark AR v86+ by Balraj Bains 2021
+// Original author for pre v86 Bridget Walsh Clair :)
+
+
+
+// Load modules
+const Scene = require('Scene');
+const Materials = require('Materials');
+const TouchGestures = require('TouchGestures');
+const Instruction = require('Instruction');
+
+
+// Main async function pt1
+(async function() {
+    // Set the instruction to show 'Tap to change'
+    Instruction.bind(true, 'tap_to_change');
```

Comment:
Would you mind adding a small comment at the top telling people to enable custom instructions in their project capabilities? I've noticed the "capabilities step" tends to confuse beginners.

This looks great 👏